R/download.R

```diff
@@ -66,7 +66,7 @@ download_method <- function() {
 
 curl_download <- function(url, path, quiet) {
 
-  if (!pkg_installed("curl")) {
+  if (!(use_external() && pkg_installed("curl"))) {
```

Comment:
I am not sure this is needed here. We only call `curl_download()` if  `use_external()`, right?

Reply:
You are probably right